src/Alignment.cpp

```diff
@@ -372,134 +372,138 @@ int Alignment::numExposedEdges(const Graph& G1, const Graph& G2) const {
 }
 
 unsigned Alignment::multiS3Numerator(const Graph& G1, const Graph& G2) const {
-    
-    uint ret = 0;
-    vector<vector<uint> > G1EdgeList;
-    vector<vector<uint> > G2EdgeList;
-    G1.getEdgeList(G1EdgeList);
-    G2.getEdgeList(G2EdgeList);
-    Matrix<MATRIX_UNIT> G1Matrix;
-    Matrix<MATRIX_UNIT> G2Matrix;
-    G1.getMatrix(G1Matrix);
-    G2.getMatrix(G2Matrix);
-    uint node1, node2;
-    
-    if ( (MultiS3::numerator_type == MultiS3::ra_i) or (MultiS3::numerator_type == MultiS3::la_i)){
-        for (const auto& edge: G1EdgeList)
-        {
-            node1 = edge[0], node2 = edge[1];
-//            if (MultiS3::_type==1){
-//                if (G2Matrix[A[node1]][A[node2]]>=1){
-//                    ret += G2Matrix[A[node1]][A[node2]] + 1; // +1 because G1 was pruned out of G2
-//                }
-//            }else if (MultiS3::_type==0){
-//                ret += G2Matrix[A[node1]][A[node2]] + 1; // +1 because G1 was pruned out of G2
-//            }
-            if       (MultiS3::numerator_type == MultiS3::ra_i){
-                // numerator is ra_i
-                if   (G2Matrix[A[node1]][A[node2]]>=1){
-                    ret += G2Matrix[A[node1]][A[node2]] + 1; // +1 because G1 was pruned out of G2
+     
+        uint ret = 0;
+        vector<vector<uint> > G1EdgeList;
+        vector<vector<uint> > G2EdgeList;
+        G1.getEdgeList(G1EdgeList);
+        G2.getEdgeList(G2EdgeList);
+        Matrix<MATRIX_UNIT> G1Matrix;
+        Matrix<MATRIX_UNIT> G2Matrix;
+        G1.getMatrix(G1Matrix);
+        G2.getMatrix(G2Matrix);
+        uint node1, node2;
+        
+        if ( (MultiS3::numerator_type == MultiS3::ra_i) or (MultiS3::numerator_type == MultiS3::la_i)){
+            for (const auto& edge: G1EdgeList)
+            {
+                node1 = edge[0], node2 = edge[1];
+    //            if (MultiS3::_type==1){
```

Comment:
If you're pretty sure this code is no longer used, remove it rather than commenting out.